azure-pipelines/build.yml

```diff
@@ -0,0 +1,141 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+# This pipeline will be extended to the MicroBuild template
+# The Task 'NuGetCommand@2' has been converted to an output named 'NuGet push to VS feed' in the templateContext section.
+# The Task 'NuGetCommand@2' has been converted to an output named 'NuGet push to NuGet.org' in the templateContext section.
```

Comment:
Should we delete this now?

Reply:
I think we should remove it before merging, yes

Reply:
+1